assets/js/modules/analytics/dashboard/dashboard-widget-overview.js

```diff
@@ -71,11 +98,17 @@ class AnalyticsDashboardWidgetOverview extends Component {
 			averageSessionDurationChange,
 		} = overviewData;
 
+		const directTotalUsersData = get( directTotalUsers, '[0].data.totals[0].values[0]' );
+
+		if ( ! directTotalUsersData ) {
+			return null;
+		}
```

Comment:
Let's do that for this piece of code too, better to move it into `toState` so that it doesn't run on every render.

It also makes the two cases consistent in behavior, with `directTotalUsers` referencing the same value.

Reply:
Good idea! Updated in https://github.com/google/site-kit-wp/pull/1277/commits/13615c809e1fed37078bf029c7e29637ea64eb02